modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -121,18 +122,18 @@ data:
     prometheusK8s:
       remoteWrite:
       - url: "https://remote-write.endpoint"
-      tlsConfig:
-        ca:
-          secret:
+        tlsConfig:
+          ca:
+            secret:
+              name: selfsigned-mtls-bundle
+              key: ca.crt
+          cert:
+            secret:
+              name: selfsigned-mtls-bundle
+              key: client.crt
+          keySecret:
```

Comment:
`keySecret:` should be `key:` according to 
https://github.com/openshift/cluster-monitoring-operator/blob/release-4.9/pkg/manifests/config.go#L200-L211
cc: @jan--f 

another `keySecret:` in this page also should be replaced

Reply:
`keySecret` is correct here. `RemoteWriteSpec` uses `SafeTLSConfig` from prometheus-operator instead of the locally defined one.

Not sure why we have this inconsistency tbh. Without diving too deep I can't see a reason to not use `monv1.TLSConfig` in ` AdditionalAlertmanagerConfig` too.

Reply:
thanks, for this PR, `keySecret` is correct, and the doc is LGTM

Reply:
Thanks @jan--f and @juzhao. I will get this merged as-is